aptible/environment.go

```diff
@@ -0,0 +1,90 @@
+package aptible
+
+import (
+	"fmt"
+	"github.com/aptible/go-deploy/client/operations"
+	"github.com/aptible/go-deploy/models"
+	"github.com/go-openapi/swag"
+)
+
+type Environment struct {
+	Deleted bool
+	Handle  string
+	ID      int64
+}
+
+type EnvironmentUpdates struct {
+	Handle string
+}
+
+type EnvironmentCreateAttrs struct {
+	Handle string
+}
+
+func (c *Client) CreateEnvironment(organizationID string, stackID int64, attrs EnvironmentCreateAttrs) (Environment, error) {
+	stack, err := c.GetStack(stackID)
+	if err != nil {
+		return Environment{}, err
+	}
+	environmentType := "production"
+	if stack.isShared() {
+		environmentType = "development"
+	}
+	params := operations.NewPostAccountsParams().WithAppRequest(&models.AppRequest{
+		Handle:         &attrs.Handle,
+		OrganizationID: &organizationID,
+		Type:           &environmentType,
+	})
+	environment, err := c.Client.Operations.PostAccounts(params, c.Token)
```

Comment:
The StackID needs to be provided in the params. I'm not sure how the tests are passing without it.

Reply:
Oh the sandbox stack is marked `default: true` so all development environments default to it when a StackID isn't provided.

Reply:
This is an excellent point and my lack of understanding of some of the managed paltform api, updated